deepchem/models/tests/test_overfit.py

```diff
@@ -645,7 +646,7 @@ def test_DAG_singletask_regression_overfit(self):
         mode="regression")
 
     # Fit trained model
-    model.fit(dataset, nb_epoch=50)
+    model.fit(dataset, nb_epoch=1200)
```

Comment:
If it worked before with 50 epochs, and now it needs 1200, it's a pretty good bet something is broken!

Reply:
I checked the original paper, and they say they trained for 5000 epochs to converge on the Delaney dataset. I did a few passes over the code, and I'm not seeing anything that's jumping out as wrong after a few hours of scrutiny. It's definitely possible something is broken here, but I'm hitting a wall on this one.